pkg/credentialquery/credentialquery.go

```diff
@@ -26,8 +55,23 @@ func NewInstance(documentLoader ld.DocumentLoader) *Instance {
 // Query returns credentials that match PresentationDefinition.
 func (c *Instance) Query(
 	query *presexch.PresentationDefinition,
-	contents []*verifiable.Credential,
+	opts ...QueryOpt,
 ) (*verifiable.Presentation, error) {
-	return query.CreateVP(contents, c.documentLoader, verifiable.WithDisabledProofCheck(),
+	qOpts := &queryOpts{}
+	for _, opt := range opts {
+		opt(qOpts)
+	}
+
+	credentials := qOpts.credentials
+	if len(credentials) == 0 {
+		var err error
+
+		credentials, err = qOpts.credentialReader.GetAll()
```

Comment:
This will panic if no credentials were provided and no credential reader is provided - you should add a `nil` check here and return an error telling the caller to specify an option.